pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -526,7 +526,7 @@ benchmarks! {
 		let reward_delay = <<T as Config>::RewardPaymentDelay as Get<u32>>::get() + 2u32;
 		let mut now = <frame_system::Pallet<T>>::block_number();
```

Comment:
I prefer this so the `end` definition at least feels more readable
```suggestion
		let mut now = <frame_system::Pallet<T>>::block_number() + 1u32;
```
They should be the same change though right?

Reply:
I think you're right, `now` and `end` are only used to control the number of iterations of the `while` loop. I'll adjust and test when I get a chance to recompile and test :laughing:

Reply:
By the way, I haven't taken the time to understand this benchmark as it's very complex, so make sure this actually makes sense! I just poked at this until something interesting happened :)